astropy/modeling/tabular.py

```diff
@@ -92,40 +95,51 @@ class _Tabular(Model):
 
     _id = 0
 
-    def __init__(self, points=None, lookup_table=None, method='linear',
-                 bounds_error=True, fill_value=np.nan, **kwargs):
+    def __init__(self, points=None, lookup_table=None,
+                 method='linear', bounds_error=True, fill_value=np.nan,
```

Comment:
Just out of curiosity, is this change an improvement? The max line length hasn't changed and it's now on 3 lines instead of two :)

Reply:
No it's not intentional. I had to break the lines in an earlier attempt making them Parameter but had to undo it but forgot to undo line break. I can undo this in next commit.